packages/integration-tests/src/entities.ts

```diff
@@ -0,0 +1,77 @@
+export enum Buffer {
+  greeterL2 = "./buffer/greeterL2.txt",
+  executeGreeterTx = "./buffer/executeGreeterTx.txt",
+  NFTtoL1 = "./buffer/NFTtoL1.txt",
+  NFTtoL2 = "./buffer/NFTtoL2.txt",
+  L1 = "./buffer/L1.txt",
+  L2 = "./buffer/L2.txt",
+  L2deposited = "./buffer/L2deposited.txt",
+  paymaster = "./buffer/paymaster.txt",
+  paymasterDeployTx = "./buffer/paymasterDeployTx.txt",
+  paymasterTx = "./buffer/paymasterTx.txt",
+  addressMultiTransferETH = "./buffer/multiTransferETH.txt",
+  txMultiTransferETH = "./buffer/txMultiTransferETH.txt",
+  txMultiTransferCustomTokenI = "./buffer/txMultiTransferCustomTokenI.txt",
+  txMultiTransferCustomTokenII = "./buffer/txMultiTransferCustomTokenII.txt",
+  addressMultiCallMiddle = "./buffer/multiCallMiddle.txt",
+  addressMultiCallCaller = "./buffer/multiCallCaller.txt",
+  addressMultiCallRoot = "./buffer/multiCallRoot.txt",
+  txMultiCallMiddle = "./buffer/txMultiCallMiddle.txt",
+  txMultiCallCaller = "./buffer/txMultiCallCaller.txt",
+  txMultiCallRoot = "./buffer/txmultiCallRoot.txt",
+  txUseMultiCallContracts = "./buffer/txUseMultiCallContracts.txt",
+  txEthTransfer = "./buffer/txEthTransfer.txt",
+  txERC20Withdraw = "./buffer/txERC20Withdraw.txt",
+  txERC20WithdrawOtherAddress = "./buffer/txERC20WithdrawOtherAddress.txt",
+  txEthWithdraw = "./buffer/txEthWithdraw.txt",
+  txEthWithdrawOtherAddress = "./buffer/txEthWithdrawOtherAddress.txt",
+  txERC20Transfer = "./buffer/txERC20Transfer.txt",
+  emptyWalletPrivateKey = "./buffer/emptyWalletPrivateKey.txt",
+  emptyWalletAddress = "./buffer/emptyWalletAddress.txt",
+  failedState = "./buffer/failedState.txt",
+  customToken = "./buffer/customToken.txt",
+}
+
+export enum Logger {
+  deposit = "DEPOSIT",
+  withdraw = "WITHDRAW",
+  transfer = "TRANSFER",
+  txHashStartsWith = "0x",
+  textSeparator = "======================= ",
+  txFailedState = "FAILED STATE",
+}
+
+export enum Token {
+  CUST_Address = "0x0928008B245A76E105E02C522b5d309c0887ecA5",
+  customL2TokenName = "L2 ERC20 token",
+  customL2TokenSymbol = "L2",
+  customL2TokenDecimals = 18,
+  ETHER_PULL_Address = "0x0000000000000000000000000000000000008001",
```

Comment:
Please use the same naming convention within a single enum/object.

Reply:
updated